index.html

```diff
@@ -6,7 +6,13 @@
 
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="content-type" content="text/html; charset=UTF8">
-
+    <!--Adding meta Tag for search engine optimisation-->
+    <meta name="Description" content="We especially welcome contributions from people from groups underrepresented in free and open source software!">
```

Comment:
@jywarren will have to advice on what he would like the main description line to be but I was think maybe also adding something about the toolbox itself? Something like: A welcome page system for GitHub-based open source software projects. We especially welcome contributions from underrepresented groups and people in free and open source software!"